Knoema.Client/Knoema.Client.sln

```diff
@@ -1,8 +1,12 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
```

Comment:
Have you confirmed this with VB?

Reply:
No, but I don't have version 2012 and think version it is no problem

Reply:
No, that's going to be a problem for developers with 2012 that use this package already.
Remember it's a public package. We shouldn't change dependencies versions unless we need some new functionality specifically.
You can use VS2013 for developing, but leave all settings for 2012.

Reply:
I returned version 2012